maxprocs/maxprocs.go

```diff
@@ -71,6 +72,13 @@ func Min(n int) Option {
 	})
 }
 
+// RoundQuota controls whether the CPU quota should be rounded using ceil or floor.
+func RoundQuota(v iruntime.Rounding) Option {
```

Comment:
With the argument being an `internal` type, it'll be impossible for a caller to actually specify this.

We need to declare an analogous type out here.

```suggestion
type Rounding = iruntime.Rounding

const (
	Ceil  Rounding = iruntime.Ceil
	Floor Rounding = iruntime.Floor
)

// RoundQuota controls whether the CPU quota should be rounded using ceil or floor.
func RoundQuota(v Rounding) Option {
```

(Ideally with the documentation copied over.)

Reply:
Also, TIL that packages in the internal folder aren't exported 😅